Automattic/wp-calypso
# Account Settings: Fix Overlap with Profile Links on Mobile

#### Changes proposed in this Pull Request

Adds some padding to the bottom of the "Profile Links" box on mobile devices, so that the inline help icon no longer overlaps with buttons. 

#### Testing instructions

Ensure that there's no overlaps and no unintended changes at https://wordpress.com/me, meaning you can click all buttons. 

**Current:**

![beforedgfdfgfgdfgd](https://user-images.githubusercontent.com/43215253/50550091-796c1c00-0c61-11e9-8bdf-0ec5d39c044f.png)

![beforefsdgfdfdggdf](https://user-images.githubusercontent.com/43215253/50550096-8557de00-0c61-11e9-9b75-783fe18a2829.png)

**Intended Proposal:**

![after_sgfdgfdgfs](https://user-images.githubusercontent.com/43215253/50550104-9f91bc00-0c61-11e9-86d4-ecba997e79fd.png)

![after_dfgdfggdf](https://user-images.githubusercontent.com/43215253/50550117-bf28e480-0c61-11e9-8e00-fa25eb7257a9.png)


**After when an error appears:**

![with_an_error_dfgfddgfgdf](https://user-images.githubusercontent.com/43215253/50551126-42533600-0c74-11e9-8722-7c961276c7e7.png)





(cc @ebinnion, @dmsnell & @tyxla) 

Fixes #26257
